src/scenes/ProgressReports/Detail/ProgressReportDrawer/ProgressReportStepper.tsx

```diff
@@ -72,44 +70,11 @@ export const ProgressReportStepper = ({ step }: ProgressReportStepperProps) => {
   const { setProgressReportStep } = useProgressReportContext();
 
   return (
-    <Paper
-      elevation={4}
-      sx={{
-        marginRight: 2,
-        overflow: 'hidden',
-      }}
-    >
-      <div
-        css={{
-          display: 'flex',
-          alignItems: 'center',
-          backgroundColor: colorPalette.stepperCard.headerBackground.partner,
-        }}
-      >
-        <PeopleJoinedIcon
-          sx={{
-            backgroundColor: colorPalette.stepperCard.iconBackground.partner,
-            marginRight: 1,
-            padding: 1,
-            height: 48,
-            width: 48,
-          }}
-        />
-        <Typography
-          sx={{
-            padding: 1,
-            flexGrow: 2,
-          }}
-        >
-          Partner Steps
-        </Typography>
+    <Paper elevation={4} sx={{ mr: 2, overflow: 'hidden' }}>
```

Comment:
How come overflow hidden?

Reply:
Not anymore. There was a weird issue back when we had a header on the stepper. Now it displays properly